.github/workflows/test.yml

```diff
@@ -35,6 +35,9 @@ jobs:
        sudo apt update -y
        sudo apt-get -y install lua5.1 luarocks
        sudo luarocks install luacheck
+       cd ..
+       sudo tarantoolctl rocks install luatest
+       cd -
```

Comment:
Now we are searching for rocks in the source and build dirs. But for the workflow, both of them are equal to `/home/runner/work/test-run/`. Without this magic with paths `.rocks` would be installed to `home/runner/work/test-run/test-run`. Do we want to add conditions search in `source, ../source, source/*`?